### DISCUSSION TITLE
Something strange with any non-english prompts with alpaca/gpt4all/vicuna

### DISCUSSION THREAD

**Discussion by 4eJIoBek1** [ggml-org/llama.cpp#760] [General]
When you try to chat with these on not english language, the answer will hallucinate and on english, but if you use file as prompt, everything works fine(picrelated). This is strange.
![IMG_20230404_090641_914](https://user-images.githubusercontent.com/109795993/229702938-279e61be-0e63-4f5d-9ee7-bd66a9d37283.jpg)

**Comment by ivanstepanovftw** [comment]
Why you even bother, model answers you perfectly
